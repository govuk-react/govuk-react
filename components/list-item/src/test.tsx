import React from 'react';
import { mount } from 'enzyme';

import { ListItem } from '.';

describe('ListItem', () => {
  it('matches wrapper snapshot', () => {
    expect(mount(<ListItem>example</ListItem>)).toMatchSnapshot('list item mount');
  });
});
