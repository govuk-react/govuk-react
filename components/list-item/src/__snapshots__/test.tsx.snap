// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem matches wrapper snapshot: list item mount 1`] = `
.c0 {
  font-family: "nta",Arial,sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  font-weight: 400;
  font-size: 16px;
  line-height: 1.25;
  margin-bottom: 5px;
}

@media print {
  .c0 {
    font-size: 14px;
    line-height: 1.15;
  }
}

@media only screen and (min-width:641px) {
  .c0 {
    font-size: 19px;
    line-height: 1.3157894736842106;
  }
}

<src__ListItem>
  <li
    className="c0"
  >
    example
  </li>
</src__ListItem>
`;
